auslib/test/common/test_rules.py

```diff
@@ -0,0 +1,69 @@
+import json
+from auslib.test.common.base import CommonTestBase
+
+
+class TestPublicRulesAPI(CommonTestBase):
+    def test_get_rules(self):
+        ret = self.public_client.get("/api/v1/rules")
+        got = json.loads(ret.data)
+        self.assertEquals(got["count"], 3)
+
+    def test_get_rules_by_product(self):
+        product = "Fennec"
+        ret = self.public_client.get("/api/v1/rules?product={}".format(product))
+        self.assertEqual(ret.status_code, 200, ret.data)
+        got = json.loads(ret.data)
+        self.assertTrue(got, "Empty dict")
```

Comment:
Can you use a more verbose error message here? Something that includes both the inputs and outputs. Eg: "No rules returned when querying for product X".